hw/ip/otbn/doc/_index.md

```diff
@@ -99,7 +99,7 @@ See the documentation for [`JAL`]({{< relref "hw/ip/otbn/doc/isa#jal" >}}) and [
 The call stack has a maximum depth of 8 elements.
 Each instruction that reads from `x1` pops a single element from the stack.
 Each instruction that writes to `x1` pushes a single element onto the stack.
-An instruction that reads from an empty stack or writes to a full stack, causes OTBN to stop, raising an alert and setting the `ERR_CODE` register to `ErrCodeCallStack`.
+An instruction that reads from an empty stack or writes to a full stack, causes OTBN to stop, raising an alert and setting the `ErrBitCallStack` bit in the {{< regref "ERR_BITS" >}} register.
```

Comment:
Since you're touching this line already, can you remove the first comma to fix the grammar?